core/control-plane/control-plane-catalog/build.gradle.kts

```diff
@@ -22,7 +22,8 @@ dependencies {
     api(project(":spi:control-plane:transfer-spi"))
     api(project(":spi:control-plane:asset-spi"))
 
-    testImplementation(project(":tests:junit-base"));
+    implementation(project(":spi:common:data-address:data-address-http-data-spi"))
```

Comment:
Why is an implementation dependency (or even test implementation) dependency needed? Can't the test use a fake type?

Reply:
it requires the `HttpDataAddressSchema.HTTP_DATA_TYPE` and `HttpDataAddressSchema.BASE_URL` constants

Reply:
Ah that's right, OK